[2025-10-16T13:23:03.949+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_etl_pipeline.transform_data scheduled__2025-10-16T13:22:00+00:00 [queued]>
[2025-10-16T13:23:03.956+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_etl_pipeline.transform_data scheduled__2025-10-16T13:22:00+00:00 [queued]>
[2025-10-16T13:23:03.956+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2025-10-16T13:23:03.967+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): transform_data> on 2025-10-16 13:22:00+00:00
[2025-10-16T13:23:03.974+0000] {standard_task_runner.py:57} INFO - Started process 139 to run task
[2025-10-16T13:23:03.978+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'csv_etl_pipeline', 'transform_data', 'scheduled__2025-10-16T13:22:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpw8m36tgp']
[2025-10-16T13:23:03.978+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask transform_data
[2025-10-16T13:23:04.020+0000] {task_command.py:410} INFO - Running <TaskInstance: csv_etl_pipeline.transform_data scheduled__2025-10-16T13:22:00+00:00 [running]> on host 252b671eecea
[2025-10-16T13:23:04.093+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-16T13:22:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-16T13:22:00+00:00'
[2025-10-16T13:23:04.113+0000] {python.py:183} INFO - Done. Returned value was: {"id":{"0":1},"prix":{"0":6.65},"logdate":{"0":1760620982648}}
[2025-10-16T13:23:04.133+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=csv_etl_pipeline, task_id=transform_data, execution_date=20251016T132200, start_date=20251016T132303, end_date=20251016T132304
[2025-10-16T13:23:04.149+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-10-16T13:23:04.167+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
